modules/IsabellenhuetteIemDcr/manifest.yaml

```diff
@@ -0,0 +1,58 @@
+description: Module implements Isabellenhuette IEM-DCR power meter driver, connecting via HTTP/REST
+config:
+  ip_address:
+    description: IPv4 Address of the power meter API.
+    type: string
+    default: "192.168.60.12"
+  port_http:
+    description: HTTP-Port of the power meter API.
+    type: integer
```

Comment:
As mentioned above, just add
```
    minimum: 1
    maximum: 65535
```
so that EVerest bails out if user configures stuff not in this range.